examples/third_party/glib/BUILD.libffi.bazel

```diff
@@ -0,0 +1,26 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "meson")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(["**"]),
+)
+
+meson(
+    name = "libffi",
+    lib_source = ":all_srcs",
+    out_interface_libs = select({
+        "@platforms//os:windows": ["ffi.lib"],
+        "//conditions:default": [],
+    }),
+    out_lib_dir = select({
+        "@platforms//os:linux": "lib/x86_64-linux-gnu",
```

Comment:
This technically isn't constrained enough - it'd break on aarch64 linux for instance but it covers what is run in CI so I'm happy with this for now.